source/contracts/reporting/Auction.sol

```diff
@@ -0,0 +1,239 @@
+pragma solidity 0.4.24;
+
+import 'libraries/DelegationTarget.sol';
+import 'reporting/IAuction.sol';
+import 'libraries/Initializable.sol';
+import 'reporting/IUniverse.sol';
+import 'reporting/IReputationToken.sol';
+import 'libraries/math/SafeMathUint256.sol';
+import 'reporting/Reporting.sol';
+
+
+contract Auction is DelegationTarget, Initializable, IAuction {
+    using SafeMathUint256 for uint256;
+
+    enum AuctionType {
+        DORMANT_1,
+        DORMANT_2,
+        UNRECORDED,
+        DORMANT_3,
+        DORMANT_4,
+        DORMANT_5,
```

Comment:
Would love some better names for these than `DORMANT_X`.

Reply:
Suggest!

Reply:
I didn't actually know what they were for, so I didn't have a better name (and why I thought they should get a better name).

Perhaps `DAY_3_NO_AUCTION`, `DAY_4_NO_AUCTION`?  Alternatively, perhaps we should just have a `day` integer and have two constants that we compare against for `UNRECORDED_AUCTION_DAY` and `RECORDED_AUCTION_DAY` rather than having an enum with a bunch of duplicate-like entries?

Reply:
I like having a specific term for the days where the auction is dormant.